src/GitHub.Api/Git/GitClient.cs

```diff
@@ -391,7 +388,46 @@ public ITask<string> AddAll(IOutputProcessor<string> processor = null)
             return last;
         }
 
-        public ITask<string> Remove(IList<string> files,
+        public ITask<string> AddAll(IOutputProcessor<string> processor = null)
+        {
```

Comment:
This method should not be moved from it's original location if it's not being changed - it will make blame commands harder in the future.